src/components/views/settings/tabs/user/AppearanceUserSettingsTab.tsx

```diff
@@ -119,6 +119,7 @@ export default class AppearanceUserSettingsTab extends React.Component<IProps, I
             );
             advanced = <>
                 <SettingsFlag name="useCompactLayout" level={SettingLevel.DEVICE} useCheckbox={true} />
+                <SettingsFlag name="showOnlyUnreadRooms" level={SettingLevel.DEVICE} useCheckbox={true} />
```

Comment:
This should probably go with the other room list settings. Possibly even be a per-sublist setting to match it up with the ordering constraints.

Reply:
I really hate the per-sublist settings.  I'd love for Design to get their teeth into the roomlist!